src/Native/System.Net.Http.Native/pal_multi.h

```diff
@@ -16,7 +16,7 @@ enum PAL_CURLMcode : int32_t
     PAL_CURLM_INTERNAL_ERROR = 4,
     PAL_CURLM_BAD_SOCKET = 5,
     PAL_CURLM_UNKNOWN_OPTION = 6,
-    PAL_CURLM_ADDED_ALREADY = 7,
+    PAL_CURLM_ADDED_ALREADY = 7, // earlier libcurls don't have this error code
```

Comment:
would be nice to be more specific than just "earlier libcurls", e.g. which version was the error code introduced.